backend/redis/utils.go

```diff
@@ -502,3 +502,7 @@ func (r *Client) AcquireLock(ctx context.Context, key string, timeout time.Durat
 func (r *Client) ReleaseLock(ctx context.Context, key string) (err error) {
 	return r.redisClient.Del(ctx, key).Err()
 }
+
+func (r *Client) FlushDB(ctx context.Context) error {
+	return r.redisClient.FlushDB(ctx).Err()
```

Comment:
could we wrap this with `if util.IsDevOrTestEnv()` to make sure we can't accidentally wire the db in production?

Reply:
good idea!
a3f6197eb